test/e2e/lazy_activation_policy_test.go

```diff
@@ -197,6 +279,51 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("3. Immediate to lazy activation policy testing", func() {
```

Comment:
this case is also belong to context `1. Policy created before resource`, so we can move this case below that context.